resources/css/site.css

```diff
@@ -1,7 +1,7 @@
 @import 'tailwindcss/base';
 @import 'tailwindcss/components';
 @import 'tailwindcss/utilities';
-@import '@fortawesome/fontawesome-pro/css/all';
```

Comment:
This import is required for font-awesome. The kit that was previously set up was not working